src/qptr/lower.rs

```diff
@@ -53,7 +53,7 @@ impl<'a> LowerFromSpvPtrs<'a> {
             Ok(match layout {
                 TypeLayout::Handle(handle) => shapes::GlobalVarShape::Handles {
                     handle: handle_layout_to_handle(handle),
-                    fixed_count: Some(NonZeroU32::new(1).unwrap()),
+                    fixed_count: Some(NonZeroU32::MIN),
```

Comment:
This is probably not a good idea because this is meant to be literally `1`, like in the sense of `[T; 1]`, and `MIN` obfuscates that slightly.

Reply:
:+1:  Reverted back to explicit `1` in c2c54934e759ce3c8a0591f8c7ed9bc3374a1b4a